### DISCUSSION TITLE
jsonschema2pojo version

### DISCUSSION THREAD

**Discussion by pgalbraith** [jsonschema2dataclass/js2d-gradle#929] [General]
I see the docs mention using jsonschema2pojo v. 1.2.1, but the 6.0.0 release of this plugin pre-dates the 1.2.1 release of jsonschema2pojo, so I am suspicious.  I also notice that using the jsonschema2pojo gradle plugin when I generate inner buliders I get a staic builder() method created, but using j2sd I do not.  This was feature was comitted in August 2022 and not released until Jan 2023, so this is making me even more suspicious that j2sd is not actually using v1.2.1 of jsonschema2pojo.

https://github.com/joelittlejohn/jsonschema2pojo/issues/1396
https://github.com/joelittlejohn/jsonschema2pojo/pull/1401

**Comment by eirnym** [comment]
Hi, thank you for a question.

Documentation for 6.0.0 says, that it uses 1.1.2. Also all code is open and you can verify every commit for each plugin version released and when documentation and other files have been updated.

Unreleased version uses 1.2.1, but plugin version 6.0.0 is still compatible with 1.2.1 as API used by this Gradle plugin didn't change.  This can be leveraged by overriding `jsonschema2dataclass` and `jsonschema2dataclassPlugin` dependency configurations to set desired versions. 

Currently I'm working on to bring new version of a plugin.
